src/pages/o/[orgId]/events/[eventId].tsx

```diff
@@ -33,53 +14,65 @@ import { ZetkinEvent } from '../../../../interfaces/ZetkinEvent';
 import { ZetkinOrganization } from '../../../../interfaces/ZetkinOrganization';
 
 const scaffoldOptions = {
-    localeScope: [
-        'misc.publicHeader',
-        'pages.orgEvent',
-    ],
+    localeScope: ['misc.publicHeader', 'pages.orgEvent'],
 };
 
-export const getServerSideProps : GetServerSideProps = scaffold(async (context) => {
-    const queryClient = new QueryClient();
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const { orgId, eventId } = context.params!;
-    const { user } = context;
+export const getServerSideProps: GetServerSideProps = scaffold(
+    async (context) => {
+        const queryClient = new QueryClient();
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        const { orgId, eventId } = context.params!;
+        const { user } = context;
```

Comment:
As discussed separately, formatting/style changes like these should not be committed. In this case, it meant that this entire function has been changed even though there were no substantial updates to logic or functionality.

In the future, when a developer wonders about the history of something in this function, they will likely use `git blame`. All lines will then indicate that this was the most recent commit, and that developer will likely spend some time trying to figure out what actually changed, before realizing that nothing changed.

When they do, they can go back and do `git blame --ignore-rev` for this commit, so no history has been lost. It's just slightly more time-consuming to follow along.